src/osdmavlink.c

```diff
@@ -306,13 +317,32 @@ void parseMavlink(void) {
   }
 }
 
+void copyNewMavlinkValuesToAirlock() {
+    // Mavlink is presumed to be the slower thread, and more tolerant of delays. Therefore
+    // we use a large timeout for the Mutex.
+    copy_osd_state_thread_safe(&mavlink_osd_state, &airlock_osd_state, portMAX_DELAY);
```

Comment:
not sure if we need the `_thread_safe` I think I get the reasoning, but if it needs to be thread safe (and there is no non-thread-safe version) I think we can get rid of it since if it is the only way to copy it the consumer of the code shouldn't have to care (imo)